Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,84 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            //Set variable
+            bool backtotop = true;
+            string option;
+
+            //Start While loop
+            while (backtotop == true)
+
+            {
+                backtotop = true;
+
+                //USer data input
+
+                Console.WriteLine("What is your first name?");
+                var firstname = Console.ReadLine();
+
+                Console.WriteLine("What is your second name?");
+                var secondname = Console.ReadLine();
+
+                Console.WriteLine("What is your age?");
+                var age = Console.ReadLine();
+
+                Console.WriteLine("What is your weight in kg?");
+                var weight = Console.ReadLine();
+
+                Console.WriteLine("What is your height in Meters?");
+                var height = Console.ReadLine();
+
+
+                //Convert vars
+                var weightsingle = Convert.ToSingle(weight);
+                var heightsingle = Convert.ToSingle(height);
```

Comment:
It would be appropriate to explicitly use float (so no var) because the method Convert.ToSingle does not indicate this properly. However, it is correct so you may stick with it.